source/content/pantheon-yml.md

```diff
@@ -253,6 +253,15 @@ Complete the following before deploying `filemount` (**required**):
 
 1. Configure a `private` subdirectory of the new path within [`protected_web_paths`](#protected-web-paths).
 
+### Search-Replace for WordPress Multisite (Early Access)
+
+You can opt-in to search-replace on [WordPress Multisites](/guides/multisite) by adding the following to `pantheon.yml`:
```

Comment:
```suggestion
You can opt-in to platform search-replace on [WordPress Multisites](/guides/multisite) by adding the following to `pantheon.yml`:
```

Should we prepend `search-replace` here?

Reply:
```suggestion
You can opt-in to platform search & replace on [WordPress Multisites](/guides/multisite) by adding the following to `pantheon.yml`:
```